src/Kernels/DynamicRupture.cpp

```diff
@@ -308,6 +318,11 @@ void seissol::kernels::DynamicRupture::flopsGodunovState( DRFaceInformation cons
   o_nonZeroFlops += dynamicRupture::kernel::computeTractionInterpolated::NonZeroFlops;
   o_hardwareFlops += dynamicRupture::kernel::computeTractionInterpolated::HardwareFlops;
 
+  o_nonZeroFlops += dynamicRupture::kernel::computeSquaredNormSlipRateInterpolated::NonZeroFlops;
+  o_hardwareFlops += dynamicRupture::kernel::computeSquaredNormSlipRateInterpolated::HardwareFlops;
+  o_nonZeroFlops += 2*tensor::squaredNormSlipRateInterpolated::size();
```

Comment:
2 or 3? + * and sqrt?

Reply:
Good question. @sebwolf-de , @ravil-mobile what do you do? We oculd count sqrt as a single floating point operation. This undercounts, but defining how many flops a sqrt takes is compilcated (and platform dependent)

Reply:
just found that on the web:
>A square root operation can take about 6 flops on typical CPU architectures, but [following [TBI97]], we will treat it as a single flop.
https://quick-adviser.com/how-do-you-calculate-number-of-flops/

Reply:
We do not count flops for the output (yet), the flops for the fortran output have not been counted either.
I'd count sqrt as 1 FLOP.